pkgs/development/tools/jo/default.nix

```diff
@@ -0,0 +1,29 @@
+{stdenv, fetchurl}:
+let
+  s = # Generated upstream information
+  rec {
+    baseName="jo";
+    version="0.9";
+    name="${baseName}-${version}";
+    url="https://github.com/jpmens/jo/releases/download/v${version}/${baseName}-${version}.tar.gz";
+    sha256="07r4m9g8696gvgz9z9m2zk83nc1ifa5siiyqplp6ijg5d5sd2mz7";
+  };
+  buildInputs = [];
+in
+stdenv.mkDerivation {
+  inherit (s) name version;
+  inherit buildInputs;
+  src = fetchurl {
+    inherit (s) url sha256;
+  };
+
+  enableParallelBuilding = true;
+
+  meta = {
+    inherit (s) version;
+    description = ''A small utility to create JSON objects'';
```

Comment:
Just a unimportant nitpick: This isn't a multi-line string nor are there any escaping issues so you can use normal double quotes (") here as well.